.vscode/tasks.json

```diff
@@ -102,7 +112,7 @@
         "command": "./build.sh",
         "type": "shell",
         "windows": {
-          "command": "./build.cmd"
+          "command": "${workspaceFolder}/build.cmd"
```

Comment:
Why the differences between workspaceFolder and workspaceRoot?

Reply:
Actually, I think these should all be workspaceFolder. https://code.visualstudio.com/docs/editor/variables-reference#_why-isnt-workspaceroot-documented

Reply:
Whoops. The inconsistency was just that this was done in two parts, over two days, on two machines, and I picked a different thing to copy and paste each time it seems.

Great catch on the `workspaceRoot` being deprecated though, I'll update all of the existing uses too.